### PR TITLE
Define Pin Colors

### PR DESCRIPTION
**Pull request by fsargent**
The loop was always evaluating NOT OUSD before getting to the Alternate. Now it's a standard IF, ELSE IF, ELSE loop with Yellow being the catch all.

### CODE REVIEW

**Comment by max-mapper** [comment]
the previous version worked correctly though, right?

**Comment by spjika** [comment]
Map is broken now actually- re Max, it did seem to work fine before- it showed Alternate correctly.

Not sure how to revert in GitHub- where to from here?

**Comment by max-mapper** [comment]
have a fix locally but waiting for github pages to come back up (theyre having issues right now apparently) https://status.github.com/

**Comment by spjika** [comment]
bahstids!
thx

Spike
openoakland.org
www.stealingbeautyphotography.com

File: source/index.js [hunk 1]
```diff
@@ -23,8 +23,14 @@ $(function() {
     if (err) return alert(JSON.stringify(err))
     data.map(function(item) { 
       item.markerURL = "images/red-dot.png"
-      if (item.Type !== 'OUSD') item.markerURL = "images/yellow-dot.png"
-      if (item.Type === 'Alternate') item.markerURL = "images/green-dot.png"
+      if (item.Type === 'OUSD' {
```
Comment (fsargent) [inline source/index.js:26]: Whoops.

**Pull request merged**